.gitignore

```diff
@@ -1,4 +1,3 @@
 .history/
 .vscode/
 node_modules/
-lib/
```

Comment:
Why did you removed lib from .gitignore? We do not want to store generated files.
Sorry, I cannot accept a PR adding generated files to git.

Reply:
I'm new ib GitHub... Can I do PR from cpecific branch? Unfortunately, I need to have the generated files in master to use my repo as a dependency.

Please, tell me how I can solve this problem correctly.